wordpress-mobile/WordPress-iOS
# [Insights Management] Replace Customize card image

Ref initial issue: #12244 
Ref conversation: https://github.com/wordpress-mobile/WordPress-iOS/pull/12659#issuecomment-543157829

To test:
- Start with a fresh install to show the Customize card.
- Go to Stats > Insights.
- Verify the shiny new image appears.


|![image_light](https://user-images.githubusercontent.com/1816888/67037295-7b14ac80-f0da-11e9-883f-a1081c14dcea.png)|![image_dark](https://user-images.githubusercontent.com/1816888/67037303-7fd96080-f0da-11e9-990f-dddd8a92ebdf.png)|
|--------|-------|

---
Update release notes:
